docs-devsite/firestore_.firestoredataconverter.md

```diff
@@ -106,15 +108,20 @@ class Post {
   }
```

Comment:
I'll add a comment that `Post` is the "app model". I'd prefer not to rename it to `PostAppModel` because nobody will call their class "BlahAppModel" in the real world :)